install.sh

```diff
@@ -951,7 +951,6 @@ metadata:
 rules:
   - apiGroups: [ "" ]
     resources:
-      - namespaces
```

Comment:
when `CLUSTER_SCOPED` is true, still need the privileged to list namespace

Reply:
I will fix it. 🙈